back-end/controllers/cloudinary.js

```diff
@@ -0,0 +1,27 @@
+const Cloudinary = (req, res) => {

```

Comment:
controller names should be verbs e.g. signin, fetchImages, etc.  So looks like this should be uploadImage?